hubs.yaml

```diff
@@ -15,6 +26,12 @@ clusters:
         template: base-hub
         auth0:
           connection: github
+          extra_allowed_users: &paleohack_users
```

Comment:
I am not sure exactly what `@paleohack_users` means, but this makes me wonder if we could just keep calling the key `allowed_users`, and then just define an explicit group for 2i2c staff that we manually add to each. Could we do something like:

```yaml
auth0:
    allowed_users: &paleohack_users &2i2c_staff
        - user1
        - user2
```

and so on?

(apologies if this question makes no sense)